spec/requests/chargebacks_spec.rb

```diff
@@ -1,5 +1,31 @@
 RSpec.describe "chargebacks API" do
   let(:field) { FactoryBot.create(:chargeable_field) }
+  let(:measure) { FactoryBot.create(:chargeback_rate_detail_measure) }
+  let(:chargeback_rate_detail) { FactoryBot.build(:chargeback_rate_detail, :description => "rate_0", :detail_measure => measure, :chargeable_field => field) }
+  let(:chargeback_tier) do
+    FactoryBot.create(:chargeback_tier, :chargeback_rate_detail_id => chargeback_rate_detail.id, :start => 0, :finish => Float::INFINITY, :fixed_rate => 0.0, :variable_rate => 0.0)
+  end
+
+  it "can fetch the list of all rates" do
+    chargeback_rate_detail.chargeback_tiers = [chargeback_tier]
+    chargeback_rate_detail.save
+
+    api_basic_authorize collection_action_identifier(:rates, :read, :get)
+    request_attributes = {:expand => 'resources', :attributes => 'chargeback_tiers,chargeback_rate,detail_measure,detail_currency,chargeable_field'}
+
+    get api_rates_url, :params => request_attributes
+
+    expect(response.parsed_body['resources'][0]['chargeback_tiers'][0]['id'].to_i).to eq(chargeback_tier.id)
+    expect(response.parsed_body['resources'][0]['chargeback_tiers'][0]['finish']).to eq('Infinity')
+    expect(response.parsed_body['resources'][0]['chargeback_rate']['id'].to_i).to eq(chargeback_rate_detail.chargeback_rate.id)
+    expect(response.parsed_body['resources'][0]['detail_measure']['id'].to_i).to eq(measure.id)
+    expect(response.parsed_body['resources'][0]['detail_currency']['id'].to_i).to eq(chargeback_rate_detail.detail_currency.id)
+    expect(response.parsed_body['resources'][0]['chargeable_field']['id'].to_i).to eq(chargeback_rate_detail.chargeable_field.id)
```

Comment:
I'm not crazy about the multitude of expects up there, can you replace with a single expect when you get a chance:

```
expect(response.parsed_body['resources'][0]).to include(
   "chargeback_tiers" => [
       {
          "id"     => chargeback_tier.id,
          "finish" => "Infinity"
       }
     ],
     "chargeback_rate" => {
          "id" => chargeback_rate_details.chargeback_rate.id
     ...
```


   

```

Reply:
I used `include` now 👍